manifests/0000_50_cluster-platform-operator-manager_00-namespace.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     release.openshift.io/feature-set: TechPreviewNoUpgrade
```

Comment:
Should these have quotes around them now that you added it to the common annotations?

Reply:
Oh that's weird that it didn't propagate to the generated manifests. I think we're fine here though given it's a no-op and we've seen successfully CI runs in #23.